src/state/slices/opportunitiesSlice/resolvers/thorchainLending/utils.ts

```diff
@@ -103,8 +102,9 @@ export const getMaybeThorchainLendingCloseQuote = async ({
 
   const { data } = await axios.get<LendingWithdrawQuoteResponse>(url)
   // TODO(gomes): handle "loan hasn't reached maturity" which is a legit flow, not an actual error
-  if (!data || 'error' in data)
-    return Err('Error fetching Thorchain lending deposit quote: no data received')
+  // i.e
+  // "failed to simulate handler: loan repayment is unavailable: loan hasn't reached maturity"
+  if (!data || 'error' in data) return Err(data.error)
```

Comment:
data might be `undefined|null|0|false|...` etc, so `return Err(data.error)` would fail in that case

Reply:
🧠 a68312e979